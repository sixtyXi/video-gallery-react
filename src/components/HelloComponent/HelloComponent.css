.paragraph {
  composes: paragraph from '../../shared/paragraph.css';
  background: lightgreen;
}
