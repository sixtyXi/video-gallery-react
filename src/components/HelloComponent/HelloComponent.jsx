import React, { Component } from 'react';

import styles from './HelloComponent.scss';

export class HelloComponent extends Component {
  render() {
    return <p className={styles.paragraph}>React Component</p>;
  }
}
