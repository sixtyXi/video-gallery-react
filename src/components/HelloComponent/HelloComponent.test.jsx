import React from 'react';
import { shallow } from 'enzyme';

import { HelloComponent } from './HelloComponent';

describe('<HelloComponent />', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<HelloComponent />);

    expect(wrapper).toMatchSnapshot();
  });
});
