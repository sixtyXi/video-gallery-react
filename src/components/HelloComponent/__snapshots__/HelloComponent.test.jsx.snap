// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloComponent /> renders correctly 1`] = `
<p
  className="paragraph"
>
  React Component
</p>
`;
